package-lock.json

```diff
@@ -1,2135 +1,8 @@
 {
```

Comment:
I think you need to restore this file to what it is on `master` and rerun. This appears to remove all `packages` from the lockfile.

Reply:
Strange it did that. Can do!